### ISSUE TITLE
Strange Header Behavior

### ISSUE THREAD

**Issue by leonardo2204** [react-native-training/react-native-elements#853]
### Description

1. I'm using header with vector-icons + TouchableHighlight. But if I click on the back-arrow, a black square appears.
2. I'm also trying to do a oneLine text + ellipsis, but it is not working.

### Reproduction Steps and Sample Code

My code:

```javascript
<Header
          outerContainerStyles={{ height: Platform.OS === 'ios' ? 70 : 70 - 24 }}
          leftComponent={<TouchableHighlight onPress={() => this.props.navigation.goBack()}>
            <View>
              <Icon name='arrow-back' iconStyle={styles.icon} />
            </View>
          </TouchableHighlight>}
          centerComponent={<Text numberOfLines={1} style={styles.barTitle}>{this.props.bloquinho.bloco_name}</Text>} />
```

Styles:

```js
icon: {
    color: 'white'
  },
barTitle: {
    color: Colors.snow,
    flexGrow: 1,
    marginLeft: 10,
  }
```

I've also tried adding `width:0` to barTitle, but the text never shows.

![header](https://user-images.githubusercontent.com/1509421/35478727-51ef7558-03cc-11e8-877f-1ceb76fad0d4.png)

Is this the expected behavior?

**Comment by xavier-villelegier** [comment]
Hey @leonardo2204,

I did [this little example](https://snack.expo.io/SJbYiVjBz) on Snack. The ellipsis is working and I used `TouchableOpacity` instead of `TouchableHighlight` to avoid this weird effect. If you really want to use `TouchableHighlight` you must set the `underlayColor` props accordingly.

![image](https://user-images.githubusercontent.com/17592779/35481808-9a7e834a-042b-11e8-9efa-38d92db6e55d.png)
